src/cascadia/TerminalCore/TerminalSelection.cpp

```diff
@@ -341,6 +342,157 @@ void Terminal::SwitchSelectionEndpoint()
     }
 }
 
+// Method Description:
+// - selects the next/previous hyperlink, if one is available
+// Arguments:
+// - searchForward: if true, we're scanning forward towards the end of the buffer. Otherwise, we're scanning backwards towards the top.
+// Return Value:
+// - true if we found a hyperlink to select (and selected it). False otherwise.
+void Terminal::SelectHyperlink(const bool searchForward)

```

Comment:
You can make this API more descriptive if you introduce a tiny `enum class` which has a `Forward` and `Backward` member.